.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Code-owners for Uptycslabs kspm-helm-charts public repository.
+
+* @gdandu-uptycs @rtammana-uptycs @nhande-uptycs @hdsingh-uptycs @abhinavm-uptycs
```

Comment:
All codeowners require write permissions to the repository. Fix `abhinavm-uptycs` permissions to the repository. Ensure `abhinavm-uptycs` got write permissions.

Reply:
`abhinavm-uptycs` is currently not a member of the uptycslabs org, adding him will automatically grant write permissions to this repo. Let's keep this file as it is & the issue will be resolved once he is added. So resolving this conversation for now.